okhttp/src/main/java/okhttp3/CertificatePinner.kt

```diff
@@ -317,29 +357,7 @@ class CertificatePinner internal constructor(
     internal fun X509Certificate.toSha1ByteString(): ByteString =
         publicKey.encoded.toByteString().sha1()
 
-    internal fun X509Certificate.toSha256ByteString(): ByteString =
+    fun X509Certificate.toSha256ByteString(): ByteString =
```

Comment:
Rename to sha256Hash and make the other one public as sha1Hash? That's as close as we can get to the Pin API.